### PR TITLE
Refine description

### PR DESCRIPTION
**Pull request by henryyang42**
Refine description
- `submit 頁面 可以不要用 Pid 嗎` -> Problem ID
- `應該是Will not be activated!!` -> will expire after...
- `在 submit 那頁加上 compiler 版本和系統版本` ->

### CODE REVIEW

**Comment by nickelcheng** [comment]
Is it possible to check system version and compiler version automatically?
Or just let user fill in it when installing.
It's not proper to hardcode it.

**Comment by henryyang42** [comment]
OK, I will add it to install

File: install.py [hunk 1]
```diff
@@ -40,26 +40,39 @@
...
+        user=raw_input('Mysql user: '),
+        password=getpass.getpass('Mysql user password: ')
+    )
+    print '========================================'
+
+if not config.has_section('system_version'):
+    # Getting system version info
+    write_config(config, 'system_version',
+        backend=raw_input('Host os version: '),
+        gcc=raw_input('gcc version: '),
```
Comment (henryyang42) [inline install.py]: get compiler version here

Comment (nickelcheng) [inline install.py]: system version is also needed

File: users/forms.py [hunk 2]
```diff
@@ -36,10 +35,15 @@
...
+    BACKEND_VERSION = config_info.get_config('system_version', 'backend')
+    GCC_VERSION = config_info.get_config('system_version', 'gcc')
+    GPP_VERSION = config_info.get_config('system_version', 'gpp')
 
-    pid = forms.CharField()
+    pid = forms.CharField(label='Problem ID')
     language = forms.ChoiceField(choices=LANGUAGE_CHOICE, initial=Submission.CPP,
-                                 widget=forms.RadioSelect())
+                                 widget=forms.RadioSelect(),
+                                 help_text="Backend: %s<br>gcc: %s, g++: %s"
```
Comment (nickelcheng) [inline users/forms.py]: how about "gcc: %s<br>g++:%s"
screenshot from 2015-04-26 22 58 11

Comment (henryyang42) [inline users/forms.py]: ＯＫ
I didn't expect the compiler version would be that long.

Comment (henryyang42) [inline users/forms.py]: The ``Ubuntu`` seem to appear many times, will we eliminate the version to ``4.7.3``?

**Comment by nickelcheng** [comment]
+1, @bruce3557 please check

**Comment by bruce3557** [comment]
Btw, need to check with the problem the student of advanced programming mentioned.
If ok, +1 with this PR

**Comment by nickelcheng** [comment]
I think the problem is about backend, @CHIASHINHOU fixed it at somewhere.
The os and compiler information has been added in this PR.

**Comment by nickelcheng** [comment]
going to merge.

File: users/forms.py (hunk 2 above)
Comment (nickelcheng) [inline users/forms.py]: It will decided by users (admin of the site)
In our OJ, we can remove it.

**Pull request merged**